Refit.Tests/InterfaceStubGenerator.cs

```diff
@@ -1,18 +1,16 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Net.Http;
-using System.Text;
 using System.Threading.Tasks;
+
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Xunit;
-using Nustache;
-using Nustache.Core;
-using Refit; // InterfaceStubGenerator looks for this
```

Comment:
Are the items in this file going to be skipped without this line? The generator looks for the `Using Refit` statement.

Reply:
Mea culpa - I have my editor set up to remove/sort using statements on save and I didn't notice the comment on that line.  I'll re-add it.